### ISSUE TITLE
[Tarea 1] Clean Break

### ISSUE THREAD

**Issue by Pdesolmi** [IIC2113-2022-2/Syllabus#18]
clean break: "If played from your hand, may reverse the card titled Jockeying for Position. Opponent must discard 4 cards. End your opponent's turn. Draw 1 card."

esas 4 cards que se descartan, es sacar las primeras 4 del arsenal y ponerlas en el ringside, sacar 4 cartas de la mano al ringside y elegirlas, o que es?

y debido al Draw 1 card en teoría en ese turno robo dos cartas? o se refiere a la carta que robaría por que me toca

Gracias!

**Comment by Pdesolmi** [comment]
Perdón, me di cuenta que puedo comprobar que es lo correcto viendo la linea 140 de la pauta en reversals

**Comment by Pdesolmi** [comment]
Al parecer las elige de la mano, pero feliz si me confirman

**Comment by RodrigoToroIcarte** [comment]
Sí. Siempre que un efecto habla de "discard" se refiere a descartar cartas de la mano.
